src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,23 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
+        [Fact, PlatformSpecific(PlatformID.FreeBSD)]
+        public void CheckFreeBSD()
         {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
+            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
+            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("FreeBSD")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("freebsd")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("bsd")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("BSD")));
             Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
             Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
         }
 
         [Fact, PlatformSpecific(PlatformID.Linux)]
         public void CheckLinux()
         {
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
I know you're trying to maintain alphabetical order here, but I think it makes more sense to have the things that are valid come first, then the things that aren't valid. Same goes for the OSX and Windows tests.